UPGRADE.md

```diff
@@ -2,6 +2,11 @@
 
 ## dev-develop
 
+## Search
+
+- Adapter namen changed e.g. from `massive_search_adapter.<adaptername>` to just `<adaptername>` in
```

Comment:
`Adapater names ...`, and I would not use a list here, since we stopped that for some time now.